packages/manager/modules/vps/src/dashboard/migrate/migrate-confirm/migrate-confirm.routing.js

```diff
@@ -1,14 +1,20 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('vps.detail.dashboard.migrate.confirm', {
     url: '/migrate',
+    params: {
+      selectedPlan: null,
+    },
     views: {
       modal: {
         component: 'ovhManagerVpsDashboardMigrateVpsConfirm',
       },
     },
     resolve: {
       migrationConfirmTrackingPrefix: /* @ngInject */ (vpsMigration) => {
-        return `vps::vps-migration-confirmation::from_${vpsMigration.currentPlan}_to_${vpsMigration.newPlan}`;
+        return `vps::vps-migration-confirmation::from_${vpsMigration.plans[0].currentPlan}_to_${vpsMigration.plans[1].newPlan}`;
```

Comment:
`plans[1]` could be null ?
Why using `plans[1]` ? I think you have to use a find method to select right plan.

Reply:
Added the suggested changes to the PR.